README.md

```diff
@@ -74,6 +74,8 @@ For full feature set have a look at prop definitions of [RecyclerListView](https
 Please go through DataProvider, LayoutProvider and rowRenderer in detail. Those are the basic building blocks of this component. For more details dive into
 the source code, lot of relevant comments have been added to help you out. 
 
+## Other projects:
```

Comment:
Should be at the bottom of the file. Also, this is too simplistic and after few swipes things don't load. I don't see it adding value here. Thoughts?

Reply:
I hard coded data for 3 months (because I was lazy) yesterday.
Here's the snack with 2 years of months. Someone can improve the logic and make the data populated whenever a month changes.

https://snack.expo.io/S1OeHVAQM